content/about/publications/index.md

```diff
@@ -39,9 +45,9 @@ papers:
   - title: "Smooth Multi-Period Forecasting with Application to Prediction of COVID-19 Cases"
     image: smoothing-paper-teaser.jpg
     authors: Tuzhilina, Hastie, McDonald, Tay, Tibshirani
-    link: https://arxiv.org/abs/2202.09723
-    journal: arXiv
-    year: 2022
+    link: https://doi.org/10.1080/10618600.2023.2285337
+    journal: Journal of Computational and Graphical Statistics
+    year: 2024
```

Comment:
do you want to move this to the top to preserve the reverse chronological ordering by publication date, or leave it here because of the original arxiv date?

Reply:
I'm fine either way. These things take so long that it might be annoying to constantly bump stuff to the top. But if someone else likes the chronology, I don't mind.

Reply:
i think you should, both for my OCD and to highlight the fact that its been properly published.

Reply:
Done. And fixed my earlier deletion of Ananya's IJCAI paper info.

Reply:
I don't think you actually deleted anything from her paper's entry...  It was previously missing the year.  The [diff from your first commit shows an invisible change on line 8](https://github.com/cmu-delphi/www-main/pull/928/commits/b2ab7c836caaf2a21b1e72b8a91962a748a64eee#diff-99ba827144198e3f7a55872591e1acfa0217543859d91e4e4487c33327dc6440L8-R8), which i think was just something with extra trailing whitespace or a funny newline character.  We should leave the year that you added though; i created [an issue for it](https://github.com/cmu-delphi/www-main/issues/929) a few hours ago, and Ananya can update the date later if necessary.